evap/staff/tests/test_views.py

```diff
@@ -18,24 +18,18 @@
                                     UserProfile)
 from evap.evaluation.tests.tools import FuzzyInt, let_user_vote_for_evaluation, WebTestWith200Check, make_manager
 from evap.rewards.models import SemesterActivation, RewardPointGranting
-from evap.staff.tools import generate_import_filename, ImportType
 from evap.staff.forms import ContributionCopyForm, ContributionCopyFormSet, EvaluationCopyForm
+from evap.staff.tests.utils import helper_enter_staff_mode, helper_exit_staff_mode, helper_delete_all_import_files
 from evap.staff.views import get_evaluations_with_prefetched_data
 
 
-def helper_delete_all_import_files(user_id):
-    for import_type in ImportType:
-        filename = generate_import_filename(user_id, import_type)
-        try:
-            os.remove(filename)
-        except FileNotFoundError:
-            pass
-
-
 class TestDownloadSampleXlsView(WebTest):
     url = '/staff/download_sample_xls/sample.xls'
     email_placeholder = "institution.com"
 
+    def setUp(self):
+        helper_enter_staff_mode(self)
```

Comment:
I was going to suggest to extend WebTest into a StaffModeWebTest, which contains just this method, but there's also WebTestWith200Check. But we could use a mixin for that, i think (I have never used them). So what about a StaffModeMixin? It doesn't change much but it's a little bit less boilerplate. Those tests that actually implemented setUp before would then need to call super().setUp(), though, and it might be unexpected when creating a new test that overriding setUp breaks staff mode. What do you think?

Reply:
I talked about this @janno42, and we decided to go for `WebTestStaffMode` and `WebTestStaffModeWith200Check`, so no mixins. The occasional `super().setUp()` calls are fine.